packages/swapkit/helpers/src/helpers/web3wallets.ts

```diff
@@ -171,12 +172,12 @@ export const listWeb3EVMWallets = () => {
 
 export function getEIP6963Wallets() {
   const providers: EIP6963Provider[] = [];
-
+  
```

Comment:
Please run linter or check your setup - it seems like it's not removing white spaces in code

Reply:
fixed